apiCalls.js

```diff
@@ -16,7 +17,7 @@ function fetchData(endpoint) {
     .then(response => checkForError(response))  
     .catch(error => {
       console.log(error)
```

Comment:
It's possible we want to leave these `console.logs` in to demonstrate that our error are generating as expected on demo, normally we'd want to take these out of a final product, but I think there's definitely value in their inclusion for the sake of the demo